website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-import-a-project-by-git-url.md

```diff
@@ -65,7 +65,7 @@ You're all set! Once support handles your request, your project is set up and yo
 
 ### Azure DevOps
 
-To add a deploy key to an Azure DevOps account, navigate to the "SSH public keys" page in the User Settings of your Azure DevOps account.
+To add a deploy key to an Azure DevOps account, navigate to the "SSH public keys" page in the User Settings of your user's Azure DevOps account or a service user's account. If possible, we recommend using a service user to mitigate issues like a user leaving the company. 
```

Comment:
Love the sentiment here - think we can frame this differently

```suggestion
To add a deploy key to an Azure DevOps account, navigate to the "SSH public keys" page in the User Settings of your user's Azure DevOps account or a service user's account. We recommend using a dedicated service user for the integration to ensure that dbt Cloud's connection to Azure DevOps is not interrupted by changes to user permissions.
```

Reply:
That's so much better. Thank you!